.github/workflows/ci.yml

```diff
@@ -21,11 +21,11 @@ jobs:
     steps:
       - name: Check out repository code
         uses: actions/checkout@v3
-      - name: Install Java 11
+      - name: Install Java 17
         uses: actions/setup-java@v3
         with:
           distribution: 'zulu'
```

Comment:
Tip unrelated to the Robo bump, but related to Java bump:
If you use [temurin](https://github.com/actions/runner-images/blob/main/README.md#package-managers-usage) distribution, then it's already installed so it'll take 0 seconds to "download" it.